plugin/rewrite/README.md

```diff
@@ -179,6 +179,16 @@ follows:
 rewrite [continue|stop] name regex STRING STRING answer name STRING STRING
 ```
 
+When using `exact` name rewrite rules, answer gets re-written automatically,
```

Comment:
IMO, we should also do this for regex matches.

Reply:
> IMO, we should also do this for regex matches.

@chrisohaver , sure. It is a bit trickier.  If there is "answer", then don't do it automatically?